kolibri/plugins/coach/assets/src/app.js

```diff
@@ -88,10 +88,11 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.RECENT_LEARNER_ITEM_DETAILS,
-          path: '/:classId/recent/:channelId/:contentId/:userId',
+          path: '/:classId/recent-learner-item-details/:userId/:channelId/item/:contentId/:questionNumber?/:interactionIndex?',
           handler: (to, from) => {
             reportsActions.showRecentLearnerItemDetails(
-              store, to.params.classId, to.params.channelId, to.params.contentId, to.params.userId
+              store, to.params.classId, to.params.userId, to.params.channelId, to.params.contentId,
+                Number(to.params.questionNumber), Number(to.params.interactionIndex)
```

Comment:
It would appear so.

My preference here is to use a redirect, rather than optional parameters, which would help avoid this, and some other logic in the action.